pkgs/top-level/all-packages.nix

```diff
@@ -25503,6 +25503,8 @@ in
 
   kubernetes-helmPlugins = dontRecurseIntoAttrs (callPackage ../applications/networking/cluster/helm/plugins { });
 
+  chart-testing = callPackage ../applications/networking/cluster/helm/chart-testing.nix { };
```

Comment:
```suggestion
  chart-testing = callPackage ../applications/networking/cluster/helm/chart-testing { };
```

Reply:
it is a `.nix` file not directory with `default.nix`

Reply:
That's the issue